fv3core/stencils/nh_p_grad.py

```diff
@@ -27,7 +27,7 @@ def calc_u(
     rdx: FloatFieldIJ,
     dt: float,
 ):
-    with computation(PARALLEL), interval(...):
+    with computation(PARALLEL), interval(0, -1):
```

Comment:
If you are going to make this change, you should also change the k range that is called, otherwise this breaks validation

Reply:
replace
`self.domain_full_k = grid.domain_shape_compute(add=(1, 1, 0)) 
self.u_domain = grid.domain_shape_compute(add=(0, 1, 0))      
  self.v_domain = grid.domain_shape_compute(add=(1, 0, 0))`
with
`self.domain_full_k = grid.domain_shape_compute(add=(1, 1, 1)) 
self.u_domain = grid.domain_shape_compute(add=(0, 1, 1))      
  self.v_domain = grid.domain_shape_compute(add=(1, 0, 1))`